application/actions/app.go

```diff
@@ -198,6 +198,7 @@ func App() *buffalo.App {
 		entityCodesGroup.GET("", entityCodesList)
 		entityCodesGroup.PUT(idRegex, entityCodesUpdate)
 		entityCodesGroup.GET(idRegex, entityCodesView)
+		entityCodesGroup.POST("/", entityCodesCreate)
```

Comment:
```suggestion
		entityCodesGroup.POST("", entityCodesCreate)
```
Not sure what difference this makes, but line 198 has an empty string, so it's probably better to be consistent.